cpp/tokens-fungible/RootTokenContract.cpp

```diff
@@ -86,8 +86,6 @@ class RootTokenContract final : public smart_interface<IRootTokenContract>, publ
     auto value_gr = int_value();
     tvm_rawreserve(std::max(start_balance_.get(), tvm_balance() - value_gr()), rawreserve_flag::up_to);
 
-    require((pubkey == 0 and internal_owner != 0) or (pubkey != 0 and internal_owner == 0),
```

Comment:
Then lines 56-57 in `deployWallet` should also be deleted:
```
 require((pubkey == 0 and internal_owner != 0) or (pubkey != 0 and internal_owner == 0),
            error_code::define_pubkey_or_internal_owner);
```
Could you, please, modify this patch?

Reply:
done this

Reply:
Thank you, merged.